core/vtk/ttkAlgorithm/vtk.module

```diff
@@ -3,4 +3,5 @@ NAME
 DEPENDS
   VTK::FiltersCore
 PRIVATE_DEPENDS
-  VTK::CommonCore
\ No newline at end of file
+  VTK::CommonCore
+  VTK::FiltersParallelDIY2 
```

Comment:
I don't believe `VTK::FiltersParallelDIY2` is needed.
if it builds and runs without it, thanks for removing it.

Reply:
Is that still needed???!!!